components/user-mgt/org.wso2.carbon.user.mgt/src/main/java/org/wso2/carbon/user/mgt/bulkimport/CSVUserBulkImport.java

```diff
@@ -133,6 +135,17 @@ public void addUserList(UserStoreManager userStore) throws UserAdminException {
         } catch (Exception e) {
             log.error(e.getMessage(), e);
             throw new UserAdminException(e.getMessage(), e);
+        } finally {
+            try {
+                if (csvReader != null) {
+                    csvReader.close();
+                }
+            } catch (IOException e) {
+                if (log.isDebugEnabled()) {
+                    log.debug(e);
+                }
+                log.error("Error occurred while closing CSV Reader");
```

Comment:
Any reason why exception is only logged if debug enabled?. Please add the reason as a comment